invokeai/frontend/web/src/features/informationalPopovers/components/clipSkip.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from 'react';
+import IAIInformationalPopover from '../../../common/components/IAIInformationalPopover';
+import { useTranslation } from 'react-i18next';
+
+export const ClipSkipPopover = (props: PropsWithChildren) => {
+  const { t } = useTranslation();
+
+  return (
+    <IAIInformationalPopover
+      paragraph={t('popovers.clipSkip.paragraph')}
```

Comment:
You could probably get rid of all of these components if you did something like this:

in the component: 
```
<IAIInformationPopover details="popovers.clipSkip" >
   ...children
</IAIInformationPopover>
```

And then in the IAIInformationPopover, update it to use the paragraph and heading based on the `details` string passed in (assuming you can interpolate the translation key which may not be possible). Then you would only have the one `details` props (and the trigger component, which could always be children) and just deduct the paragraph and heading based on that key.
```
const paragraphKey = `${props.details}.paragraph`
const headerKey = `${props.details}.heading`
if (paragraphKey && and headingKey) {
  ....use those in the JSX
}
```

Reply:
That would also make it easier to add new ones - just add the object in the translation file, and use it as a `details` prop in the IAIInformationalPopover component around whatever they want to show it

Reply:
How about also doing a Popovers.json file? Each popover is also gonna need an image, button href/label, and optional placement and idk if we should be putting all that into the translation file.